scripts/build.sh

```diff
@@ -2,39 +2,52 @@
 
 ################################################
 # Build script for LND-engine
-#
-# Params:
-# - LND_PROTO_URL (optional)
-# - INCLUDE_DOCKER (optional, defaults to false)
 ################################################
 
-set -e -u
-
-ARG=${1:-false}
+set -eu
 
 echo ""
 echo "It's time to BUILD! All resistance is futile."
 echo ""
 
-npm i
+LND_VERSION='v0.5.4-sparkswap-beta'
```

Comment:
I moved the LND_VERSION constant out of `build-images.sh` so that when this version changes, we will always have the correct proto file